clientSide/theteddyproject/src/Components/Signup/Signup.css

```diff
@@ -0,0 +1,100 @@
+.main{
+    /* border:solid 4px gray; */
+    border:solid 1px gray;
+    height:480px;
+    width:700px;
+    margin-top: 100px;
+    margin-left: 400px;
+    background-color: rgb(173, 171, 169);
+    
+}
+.mainTag{
+    display: flex;
+}
+.bottom{
+    /* border:solid 2px black; */
+    height: 50px;
+    width:696px ;
+    margin-top: -10px;
+}
+h2{
+    margin-top: 10px;
+    /* margin-left: 100px; */
+    color:white;
+    text-align: center;
+}
+.leftTag{
+
+    width:50px;
+    height: 400px;
+    
+} 
+.size{
+    height: 480px;
+    width: 350px;
+}
+.rightTag{
+ 
+    width:350px;
+    height:380px;
+    margin-left: 300px;
+}
+input{
+    padding: 10px 40px 10px 40px;
+    margin-top: 10px;
+    margin-left: 40px;
+    border-radius: 10px;
+    border:2px solid gray;
+}
+button{
+    padding: 10px 108px 10px 108px;
+    margin-top: 10px;
+    margin-left: 40px;
+    border-radius: 10px;
+    border:1px solid white;
+    background-color:gray;
+    color:white; 
+   
+}
+h6{
+    color:white;
+    font-size: 10px;
+    margin-left: 10px;
+    margin-top: 4px;
+}
+
+h5{
+    margin-left: 430px;
+    font-size: 15px;
+    margin-top: -20px;
+    color:white;
+    /* color:teal;*/
+} 
+.google{
+     height: 30px; 
+     width: 30px; 
+     margin-left: 430px;
```

Comment:
please maintain common class name for same properties

Reply:
Okay thankyou

Reply:
okay thankyou